lib/middleware.js

```diff
@@ -33,10 +33,10 @@ var checkImports = function(path, next) {
   var changed = [];
 
   nodes.forEach(function(imported){
-    fs.stat(imported.path, function(err, stat) {
```

Comment:
If the method is run it breaks with the ```imported.path```, so yes, something is not quite right there. Did you previously map each import to an object with had a path-key? I'm keeping this in the fork for now but I'll see if I can find some time to have a look at it.